config/dumps/sparql/construct_D_melanogaster.sparql

```diff
@@ -0,0 +1,12 @@
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+
+CONSTRUCT {
+	?s <http://n2o.neo/property/nodeLabel> "Drosophila_melanogaster" .
+}
+WHERE {
+  { ?s rdfs:subClassOf* obo:FBdv_00000000  } 
+  UNION { ?s rdfs:subClassOf* obo:FBdv_00007013 }
+  UNION { ?s rdfs:subClassOf* obo:FBdv_00007024 }
+  UNION { ?s rdfs:subClassOf* obo:FBdv_00005259 } 
+  }
```

Comment:
You can use VALUES instead of UNION, I think they produce the same result;
```
WHERE {
	?s rdfs:subClassOf* ?o
  values ?o {obo:FBdv_00000000 obo:FBdv_00007013 obo:FBdv_00007024 obo:FBdv_00005259 }
}
```